clusterloader2/testing/load/config.yaml

```diff
@@ -38,6 +38,15 @@ steps:
     Params:
       action: reset
 # Create RCs
+- measurements:
+  - Identifier: WaitForRunningRCs
+    Method: WaitForControlledPodsRunning
+    Params:
+      action: start
+      kind: ReplicationController
+      labelSelector: kind = loadRC
+      # operationTimeout = (totalPodsRounded /20)s + 3m
```

Comment:
Operation timeout is for a single RC - it now should depend on the size of the individual RC.

Currently we set it to 10m: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/scalability/load.go#L544
but ideally (not in this PR, but would be good to have some TODO for it somewhere) we should make it depend on how much work should be done.

Reply:
Done.